topics/admin/tutorials/job-destinations/tutorial.md

```diff
@@ -51,12 +46,23 @@ Now that you have a working scheduler, we will start configuring which jobs are
 
 {% snippet topics/admin/faqs/git-gat-path.md tutorial="job-destinations" %}
 
-# Galaxy and Slurm - Statically Mapping a Job
+# Mapping jobs to destinations
+
+In order to run jobs in Galaxy, you need to assign them to a resource manager that can handle the task. This involves specifying the appropriate amount of memory and CPU cores. For production installations, the jobs
+must be routed to a resource manager like SLURM, HTCondor, or Pulsar. Some tools may need specific resources such as GPUs or multi-core machines to work efficiently.
+
+Sometimes, your available resources are spread out across multiple locations and resource managers. In such cases, you need a way to route your jobs to the appropriate location. Galaxy offers several methods for
+routing jobs, ranging from simple static mappings to custom Python functions called dynamic job destinations.
 
-We don't want to overload our training VMs trying to run real tools, so to demonstrate how to map a multicore tool to a multicore destination, we'll create a fake tool.
+Recently, the Galaxy project has introduced a library named Total-Perspective-Vortex (TPV) to simplify this process. TPV provides a user-friendly YAML configuration that works for most scenarios.
+For more complex cases, TPV allows you to embed Python code into the YAML file. Additionally, TPV shares a global database of resource requirements, so admins don't have to figure out the requirements for each tool
+separately.
 
 ## Writing a testing tool
 
+To check that resources are being allocated appropriately, We don't want to overload our training VMs trying to run real tools, so to demonstrate how to map a multicore tool to a multicore destination using TPV,
```

Comment:
```suggestion
To demonstrate a real-life scenario and TPV's role, let's plan on setting up a configuration so that a VM designated for training jobs doesn't run and real jobs and hence doesn't get overloaded. To start, we'll create a fake tool that we'll use in our configuration.
```